packages/bkcrack/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=bkcrack
+pkgver=v1.3.0.r0.g6bcb65d
+pkgrel=1
+pkgdesc='Crack legacy zip encryption with Biham and Kocher known plaintext attack.'
+arch=('aarch64' 'x86_64')
+groups=('blackarch' 'blackarch-crackers')
+url='https://github.com/kimci86/bkcrack'
+license=('ZLIB')
+depends=('gcc-libs')
+makedepends=('git' 'cmake')
+source=("git+https://github.com/kimci86/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd "$pkgname"
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build() {
+  cd "$pkgname"
+
+  cmake .
```

Comment:
`--config Release` is [recommended by upstream](https://github.com/kimci86/bkcrack#compile-from-source)

Reply:
in order to use --config Release i will have to first generate the build directory and then the compile process,
and my code would be like this :
`
cmake -S . -B build -DCMAKE_INSTALL_PREFIX=install
cmake --build build --config Release
cmake --build build --config Release --target install
`
should i just copy these commands into the build function ? sorry new to cmake